agent/agent.go

```diff
@@ -72,7 +76,7 @@ func _main() int {
 		// All required config values can be inferred from EC2 Metadata, so this error could be transient.
 		return exitcodes.ExitError
 	}
-	log.Debug("Loaded config: %+v", *cfg)
+	log.Debug("Loaded config: ", *cfg)
```

Comment:
I think you want to change it to "Debugf" instead. See [this](http://play.golang.org/p/d_CsIFIuPb) play for the difference in information.